apps/addons/cron.py

```diff
@@ -111,6 +114,9 @@ def update_addon_average_daily_users():
 def _update_addon_average_daily_users(data, **kw):
     task_log.info("[%s] Updating add-ons ADU totals." % (len(data)))
 
+    if not waffle.switch_is_active('local-statistics-processing'):
+        return False
```

Comment:
I preferred logging first, to show we've entered the task, since it should not have been called at all when the waffle is on anyway, I prefer seeing the log